source/_components/deutsche_bahn.markdown

```diff
@@ -37,6 +37,11 @@ to:
   description: The name of the end/destination station.
   required: true
   type: string
+offset:
+  description: 
```

Comment:
missing a description 😉

Reply:
thanks. sorry, not sure why i missed that.